frontend/src/app/main/ui/notifications.cljs

```diff
@@ -8,9 +8,10 @@
   (:require
    [app.main.data.notifications :as ntf]
    [app.main.store :as st]
+   [app.main.ui.ds.notifications.toast :refer [toast*]]
    [app.main.ui.notifications.context-notification :refer [context-notification]]
    [app.main.ui.notifications.inline-notification :refer [inline-notification]]
-   [app.main.ui.notifications.toast-notification :refer [toast-notification]]
+  ;;  [app.main.ui.notifications.toast-notification :refer [toast-notification]]
```

Comment:
:fire:
Also, if toast-notification component is no longer used, it also should be removed.